libs/component-sample/src/lib/feature/side-navigation-sample/side-navigation-sample.component.html

```diff
@@ -2,17 +2,20 @@
     <sds-page-options class="sam-page__options">
         <sds-toolbar [expanded]="sideNavExpanded" #toolbar="sdsToolbar">
             <sds-accordion multi="true">
-                <sds-accordion-item [expanded]="sideNavExpanded">
+                <sds-accordion-item #navigationAccordion [expanded]="navigationExpanded">
```

Comment:
Filter isn't changing heading navigation title 
<img width="386" alt="Screen Shot 2020-06-16 at 11 22 59 AM" src="https://user-images.githubusercontent.com/63304927/84812969-15a7f100-afc4-11ea-8360-9026e039388e.png">

Reply:
I've made the update and it should work now.